parquet-column/src/main/java/org/apache/parquet/io/api/Binary.java

```diff
@@ -330,36 +354,55 @@ public static Binary fromByteArray(final byte[] value) {
   }
 
   private static class ByteBufferBackedBinary extends Binary {
```

Comment:
I seems that we can delete ByteArraySliceBackedBinary and just use this one instead with a ByteBuffer.wrap(bytes)